generate-stackbrew-library.sh

```diff
@@ -8,7 +8,7 @@ hash git 2>/dev/null || { echo >&2 "git not found, exiting."; }
 # shellcheck disable=SC2034
 array_4_8='4 argon';
 # shellcheck disable=SC2034
-array_6_11='6 boron';
+array_6_12='6 boron';
```

Comment:
Could you fix 9.x in the same pr?

Reply:
As an aside, I think we should remove minor from directories

Reply:
They provided value pre 1.0, not now

Reply:
Done, and I agree! Thanks again @SimenB 😄